dvc/command/version.py

```diff
@@ -59,12 +59,8 @@ def run(self):
                     fs_type = self.get_fs_type(repo.cache.local.cache_dir)
                     info.append(f"Cache directory: {fs_type}")
             else:
-                logger.warning(
-                    "Unable to detect supported link types, as cache "
-                    "directory '{}' doesn't exist. It is usually auto-created "
-                    "by commands such as `dvc add/fetch/pull/run/import`, "
-                    "but you could create it manually to enable this "
-                    "check.".format(relpath(repo.cache.local.cache_dir))
+                info.append(
+                    "Cache types: " + error_link("unable-to-detect-cache-type")
```

Comment:
Hi! Should some info on what the problem is be presented to the user at least? Other than just `Cache types: http://error...` Maybe I'm not seeing the full output just from this code.

Reply:
Also could we shorten the link path a bit? Maybe `"no-dvc-cache"`

Reply:
(Please see also https://github.com/iterative/dvc.org/pull/1751#pullrequestreview-480112616.)

Reply:
@jorgeorpinel Adding more info will defeat the purpose of this PR, will make the desc too long.